ranger/core/loader.py

```diff
@@ -29,7 +29,7 @@ class Loadable(object):
     paused = False
     progressbar_supported = False
 
-    def __init__(self, gen, descr, progress=ProgressTracker(0)):
+    def __init__(self, gen, descr, progress=FileProgressTracker(0)):
```

Comment:
Mutable default values in function calls are not intended in most cases because they will be shared for all invocations the default value is used. I'm confident you want to use `=None` here and check for that later.

Besides, imo that `0` can be removed since it's the default anyway.

Reply:
This indeed isn't necessary, default value replaced with `=None`.